src/game_server/handlers/zone.rs

```diff
@@ -225,10 +228,13 @@ impl From<ZoneConfig> for ZoneTemplate {
                     animation_id: transport.base_npc.active_animation_slot,
                     cursor: transport.base_npc.cursor,
                     interact_radius: transport.base_npc.interact_radius,
+                    auto_interact_radius: Some(
+                        transport.base_npc.auto_interact_radius.unwrap_or(0.0),
+                    ),
+                    move_to_interact_offset: transport.base_npc.move_to_interact_offset,
```

Comment:
```suggestion
                    auto_interact_radius: Some(
                        transport.base_npc.auto_interact_radius.unwrap_or(0.0),
                    ),
                    move_to_interact_offset: transport.base_npc.move_to_interact_offset,
```
Shouldn't this field type be `f32` instead of an `Option`, if we always assign a `Some` value?